wagtail_airtable/views.py

```diff
@@ -58,30 +58,60 @@ def get_validated_models(self):
             ]
         """
         airtable_settings = getattr(settings, "AIRTABLE_IMPORT_SETTINGS", {})
-        validated_models = []
 
+        # Loop through all the models in the settings and create a new dict
+        # of the unique settings for each model label.
+        # If settings were used more than once the second (3rd, 4th, etc) common settings
+        # will be bulked into a "grouped_models" list.
+        tracked_settings = []
```

Comment:
Probably makes a minute difference, but since your main use of this list is to check whether something's in it or not, might make more sense as a set?

Reply:
Agreed. Will change data types!